### DISCUSSION TITLE
How to use with API Resources

### DISCUSSION THREAD

**Discussion by beeyev** [flugger/laravel-responder#183] [Q&A]
Could you please give me an example, how to use this responder with [Laravel API Resources](https://laravel.com/docs/9.x/eloquent-resources) ?

**Comment by flugg** [comment]
Hi! I've been working on a completely new major version of the package which replaces Fractal with Laravel's API resources. Unfortunately, it's not 100% complete, however - it should be pretty much ready for usage. The code should be stable, but the documentation is so far incomplete. Feel free to give it a try by installing from [this](https://github.com/flugg/laravel-responder/tree/feature/api-resources) branch, and let me know what you think!

**Comment by beeyev** [comment]
That's great, I will check it out.
Thank you for your work!
